crates/rpc/Cargo.toml

```diff
@@ -26,3 +26,4 @@ ssz_types.workspace = true
 thiserror.workspace = true
 tracing.workspace = true
 warp.workspace = true
+tree_hash.workspace = true
```

Comment:
![image](https://github.com/user-attachments/assets/b0e8be3a-1093-4867-9928-02992007af78)
you re-added this